### ISSUE TITLE
Manipulable outputs

### ISSUE THREAD

**Issue by nickischenker** [DeerMaximum/Technische-Alternative-CMI#87]
Currently all nodes are integrated as binary sensors. Which at least works very well with my UVR16x2. What is annoying, however, is that the displayed values of the outputs cannot be manipulated.

It would be nice if you could not only monitor, but also switch an output on or off if necessary.

Unfortunately, at the moment this is only possible with the CMI app and cannot be automated with HA due to other factors.

I have several devices that use up residual power. However, these can only be prioritized to a very limited extent. This particularly applies to the outputs of the UVR16x2. In summer this is not so relevant, but in winter I would rather have hot water than stored electricity.

**Comment by DeerMaximum** [comment]
I don't understand your issue, can you summarize again what the error is?

**Comment by nickischenker** [comment]
Please excuse me, there is no error. I started with "Feature request - Suggest an idea for this project". Digital sensors are ideal for inputs, for outputs switches could sometimes be better. So, it's just an idea for the future that came about because I'm looking for such a solution for my energy management.

**Comment by Clark2309** [comment]
This would be a very nice feature if the CMI can be controlled from within HA. Switching the Outputs or changing fixvalues.

**Comment by DeerMaximum** [comment]
It is not possible to change values on the C.M.I. via the API used by this integration. There are unofficial methods to do this, but I will not implement them as they are experimental. As an alternative, there is my CoE integration, which can be used to send data from the Home Assistant to the C.M.I. via CAN over Ethernet.
